src/serve.ts

```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+const path = require('path')
+const app = express()
+
+app.set('port', 8080)
+
+app.use(express.static(path.join(__dirname, '../dist/')))
```

Comment:
IMO for easier development you should serve the "watched" build not the dist build.

Reply:
Where is the watched build? I was looking for something like that.